tests/TestCase/Model/Table/QueuedTasksTableTest.php

```diff
@@ -9,45 +9,58 @@
 use Cake\Core\Configure;
 use Cake\ORM\TableRegistry;
 use Cake\TestSuite\TestCase;
+use Queue\Model\Table\QueuedTasksTable;
 
+/**
+ * Queue\Model\Table\QueuedTasksTable Test Case
+ */
 class QueuedTasksTableTest extends TestCase {
 
-	/**
-	 * @var TestQueuedTask
-	 */
-	public $QueuedTask;
-
-	/**
-	 * Fixtures to load
-	 *
-	 * @var array
-	 */
+/**
+ * Fixtures
+ *
+ * @var array
+ */
 	public $fixtures = [
-		'plugin.Queue.QueuedTasks'
+		'plugin.queue.queued_tasks'
 	];
 
-	/**
-	 * Initialize the Testcase
-	 *
-	 */
+/**
+ * Initialize Tests
+ *
+ * @return void
+ */
+	public function initialize() {
+		Configure::write('App.namespace', 'TestApp');
+		$this->loadModel('Queue.QueuedTasks');
+	}
+
+/**
+ * setUp method
+ *
+ * @return void
+ */
 	public function setUp() {
 		parent::setUp();
 
-		Configure::write('App.namespace', 'TestApp');
-
-		$this->QueuedTasks = TableRegistry::get('QueuedTasks');
+		$config = TableRegistry::exists('QueuedTasks') ? [] : ['className' => 'Queue\Model\Table\QueuedTasksTable'];
+		$this->QueuedTasks = TableRegistry::get('QueuedTasks', $config);
```

Comment:
I've moved it th `initialize()` 
I'll change it back and test it without that change